main.py

```diff
@@ -2,6 +2,38 @@
 
 from django.db.models import QuerySet
 
+from db.models import Actor
+from db.models import Genre
+
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    actors = ["George Klooney", "Kianu Reaves",
+              "Scarlett Keegan", "Will Smith",
+              "Jaden Smith", "Scarlett Johansson"]
```

Comment:
The checklist specifies that values should be in a list of tuples for better iterating. Currently, the `actors` list is a list of strings. Refactoring this to a list of tuples will align your code with best practices.